plugins/CoreConsole/Commands/DevelopmentSyncProcessedSystemTests.php

```diff
@@ -34,6 +34,10 @@ protected function configure()
         $this->setDescription('For Piwik core devs. Copies processed system tests from travis artifacts to local processed directories');
         $this->addArgument('buildnumber', InputArgument::REQUIRED, 'Travis build number you want to sync, eg "14820".');
         $this->addOption('expected', 'e', InputOption::VALUE_NONE, 'If given file will be copied in expected directories instead of processed');
+        $this->addOption('repository', 'r', InputOption::VALUE_OPTIONAL, 'Repository name you want to sync screenshots for.', 'matomo-org/matomo');
```

Comment:
I guess when another repository is given we actually could skip calling `updatePluginsFiles`, as it won't be available for repos other then matomo

Reply:
added a check for this 👍